packages/teleport/src/services/user/user.ts

```diff
@@ -32,6 +37,16 @@ const service = {
     return api.get(cfg.getRequestAccessUrl(requestId)).then(makeAccessRequest);
   },
 
+  fetchAccessRequests() {
```

Comment:
Lets add a service "workflows" and move all access requests API methods in there.